resources/server/twitter/twitter.service.ts

```diff
@@ -25,60 +30,56 @@ class _TwitterService {
       // We should be able to comment this out as profile **should** be guranteed,
       // as we create a default profile in that process.
 
-      // if (!profile) {
-      //   const defaultProfileNames = await getDefaultProfileNames(identifier);
-      //   emitNet(TwitterEvents.GET_OR_CREATE_PROFILE_NULL, _source, defaultProfileNames);
-      // } else {
-      emitNet(TwitterEvents.GET_OR_CREATE_PROFILE_SUCCESS, src, profile);
-      // }
+      if (!profile) {
+        const defaultProfileNames = await getDefaultProfileNames(reqObj.source);
+
+        console.log('We have no profile');
+        console.log(defaultProfileNames);
```

Comment:
In regards to all the console logs everywhere, I don't mind implementing some further logging using our winston logger system. 

```suggestion
```